src/esp/assets/BaseMesh.h

```diff
@@ -40,6 +40,9 @@ class BaseMesh {
 
   virtual Magnum::GL::Mesh* getMagnumGLMesh() { return nullptr; }
   virtual Magnum::GL::Mesh* getMagnumGLMesh(int) { return nullptr; }
+  Corrade::Containers::Optional<Magnum::Trade::MeshData3D>& getMeshData() {
```

Comment:
Do you need this? Looks like you just want the vbo which should be accurate in `collisionMeshData_`. 🤔  I thought the original purpose of `collisionMeshData_` was to avoid public reference access to mesh data.

Reply:
By just looking at the name, `collisionMeshData_`, I was thinking it would only be used in the physics module. 

Good to know I can use it to get the geometry data. (It was really a pain point when I wrote this PR.)

Reply:
I read the code.
Actually I will finish computing the absolute AABB in the default loadScene(...). It happened even before the collision data are initialized.

> I thought the original purpose of collisionMeshData_ was to avoid public reference access to mesh data.
I have no idea about this purpose.
I guess what I can do is to make this getMeshData(...) protected.

Reply:
Checked the utility Viewer. The answer is no. We have cases that will directly call the "default" loadScene without computing the collision mesh data.

Reply:
It looks like `BaseMesh::collisionMeshData_` is not set at all for PTex scene loading (with physics or not). 

However, for other asset types such as `Mp3dInstanceMeshData`, the `collisionMeshData_` is set during the mesh load (e.g. in `Mp3dInstanceMeshData::loadMp3dPLY`) which happens in default loadScene. In this case, the optional `BaseMesh::meshData_` doesn't seem to ever be set in favor of alternative buffers (e.g. `GenericInstanceMeshData::cpu_vbo_`) which are then pointed to by the `collisionMeshData_` directly. 

Looks like a possible refactor to unify this. 🤔

Reply:
We will not do frustum culling on instance meshes.
So it is OK now that BaseMesh::meshData_ is not set for Mp3dInstanceMeshData .

Yes, we need to unify it.